style.css

```diff
@@ -0,0 +1,361 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	width: 100%;
+	max-width: 5000px;
+	margin: 20x auto; 
+}
+
+.navbar {
+	display: flex;
+	justify-content: flex-start;
+	gap: 50px;
+	padding: 15px;
+	padding-left: 45px;
+	background-color: #9a7b5c;
+	font-family: 'Roboto Flex', sans-serif;
+	font-size: 20px;
+	letter-spacing: 3px;	
+}
+
+.contact-link {
+	text-decoration: none;
+	color: #dac7b2;
+}
+
+.home-body {
+	display: flex;
+	justify-content: center;
+	min-height: 100vh;
+}
+
+.spots {
+	display: flex;
+	width: 100%;
+	object-fit: cover;
+	background-color: #dac7b2;
+}
+
+.chloe-smart {
+	position: absolute;
+	top: 15%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+	font-size: 120px;
+	letter-spacing: 30px;
+	font-family: lust-stencil,serif;
+	font-weight: 700;
+	text-align: center;
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: clip;
+}
+
+.software-developer {
+ 	position: absolute;
+  	top: 32%;
+  	left: 50.5%;
+  	transform: translate(-50%, -50%);
+	font-family: 'Roboto Flex', sans-serif;
+	letter-spacing: 48px;
+	font-size: 40px;
+  	text-align: center;
+  	white-space: nowrap;
+  	overflow: hidden;
+  	text-overflow: clip;
+}
+
+.about-me {
+	font-family: 'Roboto Flex', sans-serif;
+	font-size: 20px;
+	letter-spacing: 3px;
+}
+
+.about-me-title {
+	font-weight: 700;
+	font-size: 50px;
+	font-family: lust-stencil,serif;
+	letter-spacing: 5px;
+	padding-top: 5px;
+}
+
+.about-text {
+	padding-top: 15px;
+}
+
+.about-me-section {
+	display: flex;
+	background-color: #9a7b5c;
+	justify-content: center;
+	padding: 20px;
+	gap: 20px;
+	line-height: 40px;
+	padding-left: 50px;
+	padding-bottom: 30px;
+	padding-right: 50px;
+}
+
+.me {
+	display: flex;
+	justify-content: flex-end;
+	width: 400px;
+	height: 400px;
+	padding-top: 40px;
+	padding-bottom: 30px;
+}
+
+.portfolio-title {
+	padding-left: 45px;
+	font-family: lust-stencil,serif;
+	font-weight: 700;
+	font-size: 50px;
+	letter-spacing: 3px;
+}
+
+.project-section {
+	display: flex;
+	background-color: #cf9697;
+	flex-direction: column;
+	padding-left: 10px;
+	padding-top: 30px;
+	padding-bottom: 30px;
+	font-family: 'Roboto Flex', sans-serif;
+	line-height: 40px;
+	letter-spacing: 3px;
+}
+
+.project-1 {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+	padding: 10px;
+	gap: 50px;
+	align-items: center;
+	position: relative;
+}
+
+.project-2 {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+	padding: 10px;
+	gap: 50px;
+	align-items: center;
+	position: relative;
+}
+
+
+.project-3 {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+	padding: 10px;
+	gap: 50px;
+	align-items: center;
+	position: relative;
+}
```

Comment:
these all have literally the same styling, they should all be 1 class called `project`